packages/blockchain-wallet-v4/src/redux/payment/nfts/index.ts

```diff
@@ -21,6 +21,9 @@ import {
 
 export const cancelNftListings = async (asset: NftAsset, signer: Signer) => {
   // TODO: on front end maybe worth having a way for users to select which one of their sell orders they want to cancel and then input the order directly to this function.
+  if (!asset.sell_orders) {
```

Comment:
I think sell_orders is an array, so you may want to check !asset.sell_orders.length

Reply:
Thanks.